src/routes/clients.js

```diff
@@ -79,14 +101,28 @@ router.post('/', (req, res) => {
   });
 });
 
+const normalizeClient = client => {
+  if (client.Permissions) {
+    client.dataValues.permissions = client.Permissions.map(
```

Comment:
Yes, unfortunately we need to modify `dataValues`. Modifying `client.permissions` directly didn't work for me.